Makefile

```
@@ -1,4 +1,5 @@
 GO := go
+STATICCHECK ?= staticcheck
```

Shouldn't this be
```suggestion
STATICCHECK ?= get_staticcheck
```
?

I'm also not clear what the purpose of this change is. It appears to make the target name configurable but not much else. (Obviously not a make guru over here.)

---

So, in theory this change is to actually put a binary itself as non phony "command". This allows `make` to decide when to install it (only if the file is not there).

---

Sorry, I still don't get that.

What I'm seeing is:
1. Two `staticcheck` targets that depend on each other when the `STATICCHECK` environment variable is not set; and
2. Some arbitrarily named target (instead of a fixed `get_staticcheck`), possibly conflicting with other targets when the `STATICCHECK` environment variable is set to some value.

---

You are right. Also it was not working as you need full path to staticcheck to make it work.

You also need $GOBIN set to have any go install be meaningful (otherwise it does not know where to install), so I hope this requirement makes sense to you.

---

Fixed and tested all.